R/expect_identical_linter.R

```diff
@@ -74,17 +73,18 @@ expect_identical_linter <- function() {
     /parent::expr
   "
   expect_true_xpath <- "
-  //SYMBOL_FUNCTION_CALL[text() = 'expect_true']
-    /parent::expr
+  parent::expr
     /following-sibling::expr[1][expr[1]/SYMBOL_FUNCTION_CALL[text() = 'identical']]
     /parent::expr
   "
-  xpath <- paste(expect_equal_xpath, "|", expect_true_xpath)
-
   Linter(linter_level = "expression", function(source_expression) {
-    xml <- source_expression$xml_parsed_content
+    expect_equal_calls <- source_expression$xml_find_function_calls("expect_equal")
+    expect_true_calls <- source_expression$xml_find_function_calls("expect_true")
+    bad_expr <- c(
```

Comment:
Another `c()` inconsistent with `combine_nodesets()`

Reply:
See above, I'd only use `combine_nodesets()` if truly necessary as I expect `c()` to actually work for all use cases some time in the future.